### ISSUE TITLE
Walking to the border and getting teleported back resets essentials `/back`.

### ISSUE THREAD

**Issue by donvi-bz** [pop4959/ChunkyBorder#21]
When attempting to walk past the world border and getting teleported back, the player's /back (from essentials) gets reset.

**Comment by pop4959** [comment]
Yeah I'll have to see what I can do about this. Essentials only updates /back from other plugins when it's configured, but even if so, it kind of makes sense that running into the border shouldn't *really* count as a teleport. So while technically not a bug, it's still a problem.

**Comment by pop4959** [comment]
Implemented a workaround in 1f7ffb68291dfde0fce3342282ec6248f62059ff to prevent Essentials from running its back on teleport code when players are pushed back from the border. I've done it by temporarily adding "NPC" metadata during the teleport, which Essentials checks. This isn't an ideal solution but it's the best one I could think of without hooking directly into Essentials and forcibly setting back locations, which I wanted to avoid. I did a little preliminary digging and couldn't find any obvious incompatibilities this will cause, though it is *possible* that this logic will need to be revisited later on.
